src/Template/Aefis/xml/e2b.ctp

```diff
@@ -305,22 +309,22 @@
                 <drugrecurreadministration/>
                 <drugadditional><?php echo $aefiListOfVaccine['diluent_batch_number']; ?></drugadditional>
                 <activesubstance>
-                    <activesubstancename><?php echo $aefiListOfVaccine['vaccine_name']; ?></activesubstancename>
+                    <activesubstancename><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$aefiListOfVaccine['vaccine_name']); ?></activesubstancename>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate